Sources/SwiftInspectorAnalyzers/CachedSyntaxTree.swift

```diff
@@ -47,7 +47,7 @@ public final class CachedSyntaxTree {
   /// 
   /// - Parameter fileURL: The location of the Swift file to parse
   private func memoizeSyntaxTree(at fileURL: URL) throws -> SourceFileSyntax {
-    let cached =  try SyntaxParser.parse(fileURL)
```

Comment:
![giphy](https://user-images.githubusercontent.com/139364/106036257-ef821d80-6089-11eb-8952-e51505f1e9b0.gif)

Reply:
I really need to add linter to this project don't I 😁

Reply:
I wonder which linter we'd use 😉

Reply:
![](https://media.giphy.com/media/3diE2vbIarCXWXaN15/giphy.gif)